sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -363,6 +363,7 @@ public virtual Response<DataFeed> CreateDataFeed(DataFeed dataFeed, Cancellation
         /// <exception cref="ArgumentException"><paramref name="dataFeedId"/> is empty or not a valid GUID.</exception>
         public virtual async Task<Response> UpdateDataFeedAsync(string dataFeedId, DataFeed dataFeed, CancellationToken cancellationToken = default)
         {
+            /*
```

Comment:
Update methods will need major changes (update return type, update inputs), so commenting all of them for now. Tests disabled as well.